packages/react/ds/src/assets/logos/gov-of-ireland.tsx

```diff
@@ -1,4 +1,4 @@
-function Logo() {
+function LogoGovOfIreland() {
```

Comment:
suggestion: Can this be renamed with `GovieLog`?

Reply:
I want to keep a meaningful name, like with the file name